models/refugee.py

```diff
@@ -0,0 +1,105 @@
+from datetime import date, datetime
+
+class Refugee:
+    """
+    A class to represent a refugee family.
+    """
+    def __init__(self,
+                 name: str,
+                 camp: str,
+                 medicalCondition: str,
+                 numOfFamilyMember: int,
+                 dateOfClosing=None):
+
+        """
+        Initialise a new refugee.
+        :param name of refugee family
+        :param number of family member
+        :param camp of the refugee family locating at
+        :param medical condition of the refugee
+        :param start date of refugee family
+        :param closing date of refugee family
+        """
+
+        self.name = name
+        self.numOfFamilyMember = self.__checkNumOfFamilyMember(numOfFamilyMember)
+        self.camp = camp
+        self.medicalCondition = medicalCondition
+        self.dateOfCreation = datetime.today().date()
+        self.dateOfClosing = self.__checkDateOfClosing(dateOfClosing)
+
+    def formatName(self, name):
+        """
+        format name to a new name
+        """
+        self.name = name
+        #error handling if invalid name is inputed
+
+    def __checkNumOfFamilyMember(self, numOfFamilyMember):
+        """
+        format number of family member
+        """
+        if not isinstance(numOfFamilyMember, int) or numOfFamilyMember < 0:
+            raise self.InvalidNumOfFamilyMemberException()
+        return numOfFamilyMember + 1
+
+    def formatCamp(self, newCamp):
+        """
+        format identification camp of refugee
+        """
```

Comment:
Is the method validating the camp? If so, maybe naming it `validate_camp()` would be clearer.

Reply:
but then, I feel like validating camps should actually wait for #3, same for #25 
Therefore I suggest removing this check altogether